R/model_income_tax.R

```diff
@@ -795,9 +803,10 @@ model_income_tax <- function(sample_file,
   }
   
   switch(return.,
-         "tax" = new_tax,
-         "sample_file" = set(sample_file, j = "new_tax", value = new_tax),
-         "sample_file.int" = set(sample_file, j = "new_tax", value = as.integer(new_tax)))
+         "tax" = new_tax)
+         #"sample_file" = set(sample_file, j = "new_tax", value = new_tax),
+         #"sample_file.int" = set(sample_file, j = "new_tax", value = as.integer(new_tax)))
```

Comment:
As you can see I had to comment out these two lines as they were producing the following error which I do not understand: 
`Error in set(sample_file, j = "new_tax", value = new_tax) : 
  Internal error, please report (including result of sessionInfo()) to data.table issue tracker: oldtncol (67) < oldncol (68) but tl of class is marked.`